CHANGELOG.md

```diff
@@ -7,374 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
Shouldn't this PR add a `### Removed` section - with at least the removal of the deprecated stuff?

Also maybe a `### Changed` section with a little note about updating docs/workflow references to 4.x and PHP version to 8.3?

Reply:
Yes good point! I'll push a fixup commit with some of these details under the `[Unreleased]` heading.

Reply:
> note about updating docs/workflow references to 4.x

I don't think we need to include this in the release notes, but I will add a point about PHP 8.3.

Reply:
Done. I ended up adding the changes to each relevant commit, rather than a single fixup commit.

> I will add a point about PHP 8.3.

One other change I made is to say "8.3+" instead of just "8.3". This might require some follow-up though, to confirm that it works with PHP 8.4.

Reply:
> Change to `...3.x` in the `3.x` branch.

Done: https://github.com/farmOS/farmOS/commit/98145e2e5d9492802f342ac97f7cbac0d0542714

> Change to `3.x...4.x` in this PR.

Done: https://github.com/farmOS/farmOS/pull/949/commits/1bb19302d00c5cf073a36b681e8c3f4192579a5a#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR25

> After the first release, change to `4.0.0...4.x`.

Will do when the time comes! :smile: